ckanext/dcat/profiles.py

```diff
@@ -104,18 +104,49 @@ def _object(self, subject, predicate):
             return _object
         return None
 
-    def _object_value(self, subject, predicate):
+    def _object_value(self, subject, predicate, multilang=False):
         '''
         Given a subject and a predicate, returns the value of the object
 
         Both subject and predicate must be rdflib URIRef or BNode objects
 
         If found, the unicode representation is returned, else an empty string
         '''
+        default_lang = config.get('ckan.locale_default', 'en')
+        lang_dict = {}
         for o in self.g.objects(subject, predicate):
-            return unicode(o)
+            if multilang and o.language:
+                lang_dict[o.language] = unicode(o)
+            elif multilang:
+                lang_dict[default_lang] = unicode(o)
+            else:
+                return unicode(o)
+        if multilang:
+            # when translation does not exist, create an empty one
+            for lang in get_langs():
+                if lang not in lang_dict:
+                    lang_dict[lang] = ''
+            return lang_dict
         return ''
 
+    def _add_multilang_value(self, subject, predicate, dataset_key, dataset_dict):  # noqa
```

Comment:
It was not initially clear what this function did, would you mind adding a short docstring along the lines of "this function will add a new triple for each language defined in a multilingual field dict, eg ..."

Actually I'm wondering if the logic in this function could be incorporated to the existing helpers to add triples directly, eg [here](https://github.com/ckan/ckanext-dcat/blob/530c6d36c0889ea0301241939f82223c66c07405/ckanext/dcat/profiles.py#L530) so it is handled automatically on existing profiles without having to write your own and use `_add_multilang_value()`